### DISCUSSION TITLE
Can we convert the md file to pdf easily? Or print it?

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/introduction-to-computer-science#313] [Q&A] [answered]
So I have found this file introduction-to-computer-science/SUMMARY.md. It is a nice file because it links to all the files I need to teach the class. Unfortunately, those files are all md files. I have been looking for an md to pdf conversion. I haven't found what I would call a good one as yet. 

Does anyone have a good md to pdf converter?

Alternately is there a good way to print student handouts from an md file?

Yes, I have seen the pdf directory for each unit. No, they are not the same as the md files. If I want to use the md file version of the do now I will need to convert it or print it.

**Comment by jdonwells** [comment, marked as answer]
It looks like pandoc is the best. It will run on a Chromebook if you install it into Linux along with LaTex. The conversion is a bit small but it works.

**Reply by kenneychan** [reply]
@jdonwells you have been invited to the TEALSK12 instructor organization.  You can look at the build script for the AP CS A curriculum: https://github.com/TEALSK12/apcsa-instructor/blob/master/make.cmd, it used pandoc to generate .pdfs from .doc files.

**Comment by DBashirTEALS** [comment]
Thanks for responding to this Kenney :)
